cmd/virt-handler/base_container.cil

```diff
@@ -8,3 +8,6 @@
 (allow process cpu_online_t (file (getattr open read)))
 (allow container_runtime_t process (key (create link read search setattr view write)))
 )
+
+(allow container_t self (tun_socket (relabelto)))
+(allow container_t spc_t (tun_socket (relabelfrom)))
```

Comment:
Going forward, we are planning to remove this custom policy, hence we need to find a solution that allows virt-launcher to use this tap device while it is labeled as `container_t`

Reply:
Right, this is just here because it felt a little bit less wrong than in the `virt_launcher.cil` file . 

I'm looking for a better way of getting selinux to co-exist w/ the tap device; this is just temporary.

Reply:
Unfortunately, this doesn't make it better. This file is a base for virt_launcher.cil, if anything it should be there. However, this will soon be removed and virt-launcher will not have a special policy.